addons/danger/functions/fnc_assaultBuilding.sqf

```diff
@@ -69,8 +69,12 @@ private _building = (_buildings select 0);
 
 // find spots
 private _buildingPos = _building getVariable [QGVAR(CQB_cleared_) + str (side _unit), (_building buildingPos -1) select {lineIntersects [AGLToASL _x, (AGLToASL _x) vectorAdd [0, 0, 4]]}];
+
 private _buildingPosSelected = _buildingPos select 0;
 
+if (isNil "_buildingPosSelected") then {
```

Comment:
We can keep this in, but should never be a problem  as the findBuilding function (which assigns buildings in the first place in fn_leaderCQC.sqf) will only sort buildings with valid positions. 

I guess theoretically two units could try to remove the same position at the same time however. *shrug*

Reply:
if the variable is nil and the default is an Empty Array because it got filtered out we have to catch that.